src/UnoDrive.Shared/ViewModels/MyFilesViewModel.cs

```diff
@@ -32,6 +32,7 @@ public MyFilesViewModel(IGraphFileService graphFileService, ILogger<MyFilesViewM
 			Back = new AsyncRelayCommand(OnBackAsync);
 
 			FilesAndFolders = new List<OneDriveItem>();
+
 			this.networkConnectivity.NetworkStatusChanged += OnNetworkStatusChanged;
```

Comment:
Can this cause a memory leak or will we have just one instance of this VM during all application life cicle?

Reply:
I've been debating about introducing a WeakEventManager like we have in XCT but that becomes 1 more level of complexity for this sample code. Since I have a dependency on Windows Community Toolkit I might be able to leverage the Weak Event Handler which is a similar concept